src/data/priv_appendable_data.rs

```diff
@@ -19,10 +19,11 @@ use maidsafe_utilities::serialisation::{deserialise, serialise, serialised_size}
 use rust_sodium::crypto::{box_, sealedbox};
 use rust_sodium::crypto::sign::{self, PublicKey, SecretKey, Signature};
 use rustc_serialize::{Decodable, Decoder};
-use std::collections::BTreeSet;
+use std::collections::{BTreeMap, BTreeSet};
 use std::fmt::{self, Debug, Formatter};
 use xor_name::XorName;
-use super::{AppendWrapper, AppendedData, DataIdentifier, Filter, NO_OWNER_PUB_KEY, verify_detached};
+use super::{AppendWrapper, AppendedData, DataIdentifier, Filter, NO_OWNER_PUB_KEY,
+            verify_signatures};
```

Comment:
For functions, the policy is to import the module instead. (Probably you don't need to import anything and can just use `super::verify_signatures`.)